src/org/traccar/model/Position.java

```diff
@@ -22,9 +22,11 @@
     public static final String KEY_ORIGINAL = "raw";
     public static final String KEY_INDEX = "index";
     public static final String KEY_HDOP = "hdop";
-    public static final String KEY_SATELLITES = "sat";
+    public static final String KEY_SATELLITES = "sat"; // in use
+    public static final String KEY_SATELLITES_VISIBLE = "satVisible";
```

Comment:
Understood.  I will review the documentation of some of the other protocols to determine if this data is available.  There is utility in this data.  If it is only used here, I will "hardcode" the key as you suggest.